api.swagger.yaml

```diff
@@ -5471,6 +5471,29 @@ paths:
           description: |
             The notification does not exist.
       description: Mark a notification as read.
+  '/notifications/clear_all':
+    parameters:
+      - description: Marks all notifications for the mobile app as read. If true, this will mark as read notifications with entity type user, and entity type node and bundle type post only.
+        in: query
+        name: app_notifications_only
+        schema:
+          type: boolean
+          default: false
+    put:
+      tags:
+        - notifications
+      responses:
+        '200':
```

Comment:
And can this be a 204 response? We can remove the response body and related schema